Workbooks/Azure Active Directory TroubleShooting/ArchivedLogDateRange/ArchivedLogDateRange.workbook

```diff
@@ -0,0 +1,55 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "### Sign-in Date Range"
+      },
+      "name": "text - 2"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let oldestSignIn = SigninLogs | where TimeGenerated > ago(3653d) | order by TimeGenerated asc | limit 1 | project OldestSignInEvent=TimeGenerated,Type;\r\nSigninLogs | where TimeGenerated > ago(3653d) | order by TimeGenerated desc | limit 1 | project NewestSignInEvent=TimeGenerated,Type| join oldestSignIn on Type | project OldestSignInEvent,NewestSignInEvent\r\n",
+        "size": 4,
+        "noDataMessage": "No sign-ins recorded.",
+        "exportToExcelOptions": "visible",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "customWidth": "50",
+      "name": "signinlog query",
+      "styleSettings": {
+        "maxWidth": "50"
+      }
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "### Audit Log Date Range"
+      },
+      "name": "text - 3"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "let oldestAudit = AuditLogs | where TimeGenerated > ago(3653d) | order by TimeGenerated asc | limit 1 | project OldestAuditEvent=TimeGenerated,Type;\r\nAuditLogs | where TimeGenerated > ago(3653d) | order by TimeGenerated desc | limit 1 | project NewestAuditEvent=TimeGenerated,Type| join oldestAudit on Type | project OldestAuditEvent,NewestAuditEvent\r\n",
```

Comment:
is this the same thing but simpler? `AuditLogs | where TimeGenerated > ago(3653d) | summarize OldestAuditEvent=min(TimeGenerated), NewestAuditEvent=max(TimeGenerated) by Type`

Reply:
Thanks for the input, that works.  Updated workbook.